docs/reference/announcements-release-notes/880/whats-new-in-88.md

```diff
@@ -438,15 +438,15 @@ Camunda 8.8 includes new tools and enhancements to help you migrate from Camunda
     <td>**Description**</td>
 </tr>
 <tr>
-    <td>[Data migration tool](/guides/migrating-from-camunda-7/data-migrator/index.md)</td>
+    <td>[Data migration tool](/guides/migrating-from-camunda-7/migration-tooling/data-migrator/index.md)</td>
     <td>Use the Data Migrator to copy running process instances from Camunda 7 to Camunda 8.</td>
 </tr>
 <tr>
-    <td>[Migration Analyzer & Diagram Converter](/guides/migrating-from-camunda-7/migration-tooling.md#migration-analyzer--diagram-converter)</td>
+    <td>[Migration Analyzer & Diagram Converter](/guides/migrating-from-camunda-7/migration-tooling/index.md#migration-analyzer--diagram-converter)</td>
```

Comment:
These references aren't valid:

**index.md#migration-analyzer--diagram-converter**

I think we want to link to:

`/guides/migrating-from-camunda-7/migration-tooling/diagram-converter/`

instead.